cli/fluffy_cli/main.py

```diff
@@ -80,9 +80,16 @@ def upload(server, paths, auth, direct_link):
             print(bold(resp['redirect']))
 
 
-def paste(server, path, language, highlight_regex, auth, direct_link):
+def paste(server, path, language, highlight_regex, auth, direct_link, tee):
+    content = ''
     if path == '-':
-        content = sys.stdin.read()
+        if tee:
+            # We should stream it
+            for line in sys.stdin:
+                content += line
+                print(line, end='')  # The line will already have a newline
```

Comment:
I wonder whether it should have `flush=True` here to make sure it flushes after every print. Otherwise I think Python 3 will buffer this by default if stdout isn't a tty (I hate that new behavior).

Let me test it out...

Reply:
Yeah, I think flushing would be good. You can test with a command like `(echo hello; sleep 1; echo hello2; sleep 1; echo hello3) | fpb --tee | cat` -- you don't see any of the echos until the process exits (or you print enough to fill the buffer) which isn't great.

Probably not a big deal because people are unlikely to pipe fpb to anything, but I think it would be worth adding a `flush=True` -- I can do that!